packages/oui-popover/src/popover.controller.js

```diff
@@ -31,16 +31,31 @@ export default class PopoverController {
     }
 
     $postLink () {
-        this.setPopover();
-        this.setTrigger();
+        this.setPopover()
+            .then(() => this.setTrigger())
+            .then(() => {
+                if (this.open) {
+                    this.openPopover();
+                }
+            });
+    }
+
+    $onChanges (changes) {
+        if (angular.isDefined(changes.open) && this.triggerElement) {
```

Comment:
Didn't you want to create code as much decoupled from Angular as possible ?

```suggestion
        if (changes.open != null && this.triggerElement) {
```

or 

```suggestion
        if (changes.open !== undefined && this.triggerElement) {
```

if you are afraid of `==` :)